src/vue/common/OperationRecord.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <div class="app__card operation-record">
+    <table class="operation-record__table">
+      <tr class="operation-record__tr">
+        <td>
+          <span class="operation-record__date">
+            {{ operation.date | formatCalendar }}
+          </span>
+        </td>
+        <td>
+          {{ `operation-names.${operation.typeI}` | globalize }}
+        </td>
+        <td>
+          <span
+            :class="{'operation-record__negative-amount'
+              : accountId === operation.sender}">
```

Comment:
for such cases of ugly formatting I recommend using of `<!-- eslint-disable-next-line max-len -->`
or formatting like
```
:class="{
  'operation-record__negative-amount' : accountId === operation.sender}"
"}
```
also you can extract `accountId === operation.sender` into computed property